spec/integration/provider/package_spec.rb

```diff
@@ -12,7 +12,7 @@
         options = {:name => "nosuch#{provider.name}", :provider => provider.name}
 
         pkg = Puppet::Type.newpackage(options)
-        expect { pkg.provider.install }.to raise_error
+        expect { pkg.provider.install }.to raise_error(/source ?(parameter)?|gem|pip|lock file/)
```

Comment:
I'm guessing this returns potentially different error messages depending on the default package provider for the system where the test is run? Is the 'lock file' error coming from puppet's lock file when it's applying a catalog or one of the package providers?